src/libcommonserver/log/log.cpp

```diff
@@ -111,4 +112,10 @@ Log::Log(const log4cplus::tstring &filePath) : _filePath(filePath) {
 SyncPath Log::getLogFilePath() const {
     return _filePath;
 }
+
+void Log::checkForExpiredFiles() {
+    auto customRollingFileAppender = static_cast<CustomRollingFileAppender *>(_logger.getAppender(Log::rfName).get());
```

Comment:
```suggestion
    auto *customRollingFileAppender = static_cast<CustomRollingFileAppender *>(_logger.getAppender(Log::rfName).get());
```
Just to make clear that we retrieve a pointer.